cmd/root.go

```diff
@@ -435,15 +440,13 @@ func ensureToken(ctx context.Context, oi sdp.OvermindInstance, requiredScopes []
 // that token locally for use later, and will use the cached token if possible
 func getOauthToken(ctx context.Context, oi sdp.OvermindInstance, requiredScopes []string) (*oauth2.Token, error) {
 	var localScopes []string
-
-	// Check for a locally saved token in ~/.overmind
-	if home, err := os.UserHomeDir(); err == nil {
-		var localToken *oauth2.Token
-
-		localToken, localScopes, err = readLocalToken(home, requiredScopes)
-
+	var localToken *oauth2.Token
+	home, err := os.UserHomeDir()
+	if err == nil {
+		// Check for a locally saved token in ~/.overmind
+		localToken, localScopes, err = readLocalTokenFile(home, viper.GetString("app"), requiredScopes)
 		if err != nil {
-			log.WithContext(ctx).Debugf("Error reading local token, ignoring: %v", err)
+			pterm.Info.Println(fmt.Sprintf("Error reading local token: %v. Re authenticating.", err))
```

Comment:
After hearing you talk about this in standup, you're right it would be good to tell people this is happening and why, but just needs to be less alarming (i.e. not say error)

Reply:
made the necessary changes, and screen shotted some examples below